src/R2.jl

```diff
@@ -98,17 +101,19 @@ function SolverCore.solve!(
   σmin = zero(T),
   max_time::Float64 = 3600.0,
   max_eval::Int = -1,
+  β = T(0),
   verbose::Int = 0,
 ) where {T, V}
   unconstrained(nlp) || error("R2 should only be called on unconstrained problems.")
-
+  β = T(β)
```

Comment:
```suggestion
```

Reply:
Note that there are two different β symbols here.

Reply:
yes, I removed it since it was a typo